setup.py

```diff
@@ -16,21 +16,38 @@
     package_dir={'': 'src'},
     zip_safe=False,
     platforms='any',
-    python_requires='>=3.6',
+    python_requires='>=3.7',
     install_requires=[
         'attrs>=18.1',
         'isodate',
         'python-dateutil',
-        'rfc3986<2',  # version 2.0.0 dropped python 3.6 support.
+        'rfc3986<2',  # Pin until https://github.com/python-hyper/rfc3986/issues/86 is resolved.
         'uritemplate>=3.0.0',
+        'babel',
+        'requests',
+        'language-tags',
+        'rdflib',
+        'colorama',
```

Comment:
With dependencies like `rdflib` this package can't be called "slim" anymore, I guess. But considering the better csv support in PY3, no one would use it just for unicode in CSV files. So we go all in, paving the way for csvw2rdf sometime soon :)